extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php

```diff
@@ -48,6 +48,12 @@ static public function onWikiaSkinTopScripts( &$vars, &$scripts, $skin ){
 			$vars['wgGAUserIdHash'] = md5($user->getId() . $wgGAUserIdSalt);
 		}
 
+		global $wgDevelEnvironment, $wgStagingEnvironment;
```

Comment:
I always prefer having global in one place of function body. But it's my personal opinion :japanese_ogre: I don't think our guidelines cover it - feel free to change it or not :smiley_cat: